flipperdevices/flipperzero-firmware
# [FL-3678] NFC UI refactor

# What's new

- Filename or "Unsaved + CardType" is now showed for saved cards during emulation
- Headers added to Write scenes
- Reordered menu items according to new spec

# Verification 

- Read any card and try to emulate it without saving. Result: "Unsaved Card Type" will be printed during emulation. On saved card filename will be printed
- Try to write any NTAG, there will be a new header
- Go to saved menu choose any card and check the order of menu items

# Checklist (For Reviewer)
